src/containers/Configure/index.js

```diff
@@ -13,28 +14,64 @@ const myTheme = {
 
 export default class Configure extends Component {
 
-	handleComplete(err, confirmation) {
+	handleComplete(err, config) {
 		if (err) {
 		  alert('Configuration setup failed')
 		  return;
 		}
-	
-		alert('Success: ' + JSON.stringify(confirmation, null, 2));
+        var preference = { pomodoroSize: config.slots.pomodorosize,
+                           shortBreakSize: config.slots.shortbreaksize,
+                           longBreakSize: config.slots.longbreaksize,
+                           workSchedule: { start: {},
+                                           lunch: { start: {}, end: {} },
+                                           end: {} } };
+
+        preference.workSchedule.start = this.convertTimeString(config.slots.workstarttime);
+        preference.workSchedule.end = this.convertTimeString(config.slots.workendtime);
+        preference.workSchedule.lunch.start = this.convertTimeString(config.slots.lunchstarttime);
+        preference.workSchedule.lunch.end = this.convertTimeString(config.slots.lunchendtime);
+
+        this.setPreferences(preference).then(() => {
+            console.log("done with bot");
+        });
+
 		return 'Configuration is complete! Start adding some tasks';
```

Comment:
@navjotbola can you please take a look at what's happening here, this message is never printed in the bot ui.